test/helper.js

```diff
@@ -67,7 +67,7 @@ if (['false', 'f', '0', 'n', 'no'].indexOf(env.AO_TEST_SHOW_LOGS) >= 0) {
   ao.logLevel = 'none'
 }
 
-const BSON = new bson.BSONPure.BSON()
+//const BSON = new bson.BSONPure.BSON()
```

Comment:
delete?

Reply:
yes, thanks. it was a quick test to make sure things worked when i updated bson from v 0.4.? to v4.?. and once it worked i didn't go back. removed.